zam/aspirateur/zam_aspirateur/amendements/writer.py

```diff
@@ -0,0 +1,87 @@
+import csv
+from dataclasses import fields
+from typing import Iterable
+
+from openpyxl import Workbook
+from openpyxl.styles import (
+    Color,
+    Font,
+    PatternFill,
+)
+from openpyxl.worksheet import Worksheet
+
+from .models import Amendement
+
+
+FIELDS = [field.name for field in fields(Amendement)]
+
+FIELDS_NAMES = {
+    'article': 'Nº article',
+    'alinea': 'Alinéa',
+    'num': 'Nº amdt ou sous-amdt',
+    'auteur': 'Auteur(s)',
+    'date_depot': 'Date de dépôt',
+    'discussion_commune': 'Discussion commune ?',
+    'identique': 'Identique ?',
+}
+
+
+DARK_BLUE = Color(rgb='00182848')
+WHITE = Color(rgb='00FFFFFF')
+
+
+def write_csv(amendements: Iterable[Amendement], filename: str) -> int:
+    nb_rows = 0
+    with open(filename, 'w', encoding='utf-8') as file_:
+        writer = csv.DictWriter(
+            file_,
+            fieldnames=FIELDS,
+            delimiter=';',
+            quoting=csv.QUOTE_MINIMAL,
+        )
+        writer.writeheader()
+        for amendement in amendements:
+            writer.writerow(amendement.as_dict())
+            nb_rows += 1
+    return nb_rows
```

Comment:
Isn't it the same as `len(amendements)`?

Reply:
`amendements` is an `Iterable[Amendement]` here, so it might be an iterator, not a sequence, in which case we can only consume it once